pkg/config/config.go

```diff
@@ -139,12 +139,13 @@ type ReceiverConfig struct {
 	ReopenDuration *Duration `yaml:"reopen_duration" json:"reopen_duration"`
 
 	// Optional issue fields
-	Priority          string                 `yaml:"priority" json:"priority"`
-	Description       string                 `yaml:"description" json:"description"`
-	WontFixResolution string                 `yaml:"wont_fix_resolution" json:"wont_fix_resolution"`
-	Fields            map[string]interface{} `yaml:"fields" json:"fields"`
-	Components        []string               `yaml:"components" json:"components"`
-	StaticLabels      []string               `yaml:"static_labels" json:"static_labels"`
+	Priority             string                 `yaml:"priority" json:"priority"`
+	Description          string                 `yaml:"description" json:"description"`
+	WontFixResolution    string                 `yaml:"wont_fix_resolution" json:"wont_fix_resolution"`
+	Fields               map[string]interface{} `yaml:"fields" json:"fields"`
+	Components           []string               `yaml:"components" json:"components"`
+	StaticLabels         []string               `yaml:"static_labels" json:"static_labels"`
+	CustomFieldsToUpdate []string               `yaml:"update_always_fields" json:"update_always_fields"`
```

Comment:
Do you want only a few fields to be updated? Or an option "update_fields: true" for all fields would be better?

Also if anything, this could be probably:
 
```suggestion
	FieldsToUpdate []string               `yaml:"update_always_fields" json:"update_always_fields"`
```